Shell/LineEditor.h

```diff
@@ -32,7 +32,10 @@ class LineEditor {
     void vt_clear_to_end_of_line();
 
     Vector<char, 1024> m_buffer;
+    Vector<String> m_suggestions;
```

Comment:
Does this need to be a member variable? Can it be a return value from the suggestion function instead?

Reply:
Isn't returning a `Vector<String>` on the stack naughty?